features/step_definitions/stepdefs.js

```diff
@@ -1579,6 +1585,7 @@ When(
             const activity = await this.response.clone().json();
 
             this.activities[noteName] = activity;
+            this.objects[noteName] = activity.object;
```

Comment:
We only had access to the `Create(Note)` activity but we needed the underlying object for checking the `Delete(Note)` activity.

Maybe we wanna fix these up eventually so they have different names? Not sure

Reply:
Yeh i ended up doing this same thing in some other step